app/controllers/AppInit.go

```diff
@@ -1,26 +1,26 @@
 package controllers
 
 import (
-	"database/sql"
+	"github.com/gocql/gocql"
 	_"github.com/go-sql-driver/mysql"
-	"fmt"
 	"github.com/revel/revel"
+	"Revel-AddressBook/app/models"
+	"fmt"
 )
 type AppInit struct {
 	*revel.Controller
 }
-var Database *sql.DB
-func InitDB() {
+func InitDB(){
```

Comment:
i changed the position of the function into the AppInit model and changed the name of the variable but it has to be public so it can be accessed from other controllers .